command/daemon.go

```diff
@@ -40,7 +40,7 @@ const (
 	// configCheckInterval is the time between config update checks.
 	configCheckInterval = 30 * time.Second
 	// shutdownTimeout is the duration that a graceful shutdown has to complete.
-	shutdownTimeout = 5 * time.Second
+	shutdownTimeout = 10 * time.Second
```

Comment:
Side note: when storetheindex runs on K8S we need to make sure that the value configured here is not less than `terminationGracePeriodSeconds` specified in statefulset pod template. The default value for it is 30 seconds. 

Ultimately `terminationGracePeriodSeconds` is the value that decides how long to wait before force killing  the pod if `shutdownTimeout` is longer.